tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_BentBeam.cpp

```diff
@@ -120,25 +128,37 @@ SPECTRE_TEST_CASE(
       std::make_tuple(), DataVector(5));
 
   {
-    INFO("Test elasticity system with bent beam");
-    // Verify that the solution numerically solves the system
     using system = Elasticity::FirstOrderSystem<2>;
     const typename system::fluxes fluxes_computer{};
     using AffineMap = domain::CoordinateMaps::Affine;
     using AffineMap2D =
         domain::CoordinateMaps::ProductOf2Maps<AffineMap, AffineMap>;
     const domain::CoordinateMap<Frame::Logical, Frame::Inertial, AffineMap2D>
-        coord_map{{{-1., 1., -2.5, 2.5}, {-1., 1., -0.5, 0.5}}};
+        coord_map{{{-1., 1., -5. / 2., 5. / 2.}, {-1., 1., -1. / 2., 1. / 2.}}};
     // Since the solution is a polynomial of degree 2 it should numerically
     // solve the system equations to machine precision on 3 grid points per
     // dimension.
     const Mesh<2> mesh{3, Spectral::Basis::Legendre,
                        Spectral::Quadrature::GaussLobatto};
     const auto logical_coords = logical_coordinates(mesh);
     const auto inertial_coords = coord_map(logical_coords);
+    INFO("Test elasticity system with bent beam");
```

Comment:
Please wrap the `INFO` and the associated testing code in a scope `{}`, then the `INFO` is correctly printed when the associated testing code fails. When doing so, make sure to move all code that's associated with only one of the tests into its scope (e.g. the `using system` should be in the verify-system-test).